src/OpenProject.Revit/Entry/OpenViewpointEventHandler.cs

```diff
@@ -17,16 +17,15 @@ namespace OpenProject.Revit.Entry
   /// </summary>
   public class OpenViewpointEventHandler : IExternalEventHandler
   {
-    /// <summary>
-    /// This is the method declared in the <see cref="IExternalEventHandler"/> interface
-    /// provided by the Revit API
-    /// </summary>
-    /// <param name="app"></param>
+    private const decimal _viewpointAngleThresholdRad = 0.087266462599716m;
```

Comment:
this is 5 degrees. should we directly make it configurable? Or should we stick first with a constant value.